src/toil/test/__init__.py

```diff
@@ -48,6 +54,10 @@ class ToilTest(unittest.TestCase):
     _tempBaseDir = None
     _tempDirs = None
 
+    # Use us-west-2 unless running on EC2, in which case use the region in which
+    # the instance is located
+    AWSRegion = _region() if runningOnEC2() else 'us-west-2'
```

Comment:
Let's just make awsRegion an `@classmethod`, and the support functions too. That would keep all the related code in a single place. We'd be trading in the omission of `()` but that's an acceptable trade-off.